spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/Predictor.scala

```diff
@@ -194,6 +228,7 @@ object Predictor {
       })
     }
   }
+*/
```

Comment:
why leave it commented ?

Reply:
That is the old predictClass api. Should I remove it or keep it with a @deprecated annotation?